src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import { Input, Button } from 'buildo-react-components';
+import Dropdown from '../Dropdown';
+import View from '../View/View';
+
+const rangeOptions = [
+  { value: 1, label: '1 km' },
+  { value: 5, label: '5 km' },
+  { value: 10, label: '10 km' },
+  { value: 15, label: '15 km' },
+  { value: 25, label: '25 km' },
+  { value: 35, label: '35 km' },
+  { value: 50, label: '50 km' }
+];
+
+const SearchBar: React.FC = () => {
```

Comment:
Could simply be like this:
```suggestion
export function SearchBar() {
```